jdaviz/core/tests/test_tools.py

```diff
@@ -41,3 +42,29 @@ def test_rangezoom(specviz_helper, spectrum1d):
     t.interact.selected = [14, 15]
     t.on_update_zoom()
     assert_allclose(_get_lims(sv), [6500, 7000, 14, 15])
+
+
+def test_stretch_bounds(imviz_helper):
+    imviz_helper.load_data(np.ones((2, 2)))
+
+    plot_options = imviz_helper.plugins['Plot Options']._obj
+    print(plot_options.stretch_vmin_value)
+    print(plot_options.stretch_vmax_value)
```

Comment:
```suggestion
```

Reply:
Whoops, I always seem to miss some print somewhere ☠️